x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/rest/RestEnrichStatsAction.java

```diff
@@ -33,8 +33,10 @@ public String getName() {
     }
 
     @Override
-    protected RestChannelConsumer prepareRequest(final RestRequest restRequest, final NodeClient client) throws IOException {
-        final EnrichStatsAction.Request request = new EnrichStatsAction.Request();
+    protected RestChannelConsumer prepareRequest(final RestRequest restRequest, final NodeClient client) {
+        final var request = new EnrichStatsAction.Request(RestUtils.getMasterNodeTimeout(restRequest)
+        // NEW
```

Comment:
?

Reply:
Bah that was a note-to-self I forgot to remove

Reply:
Gone in 4f1f69b44fa.